src/applications/static-pages/widget-creators/alerts-dismiss-view.js

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
This was copy-pasted from the old alerts-dismiss-view, right? 🙂

Reply:
I actually moved it and made (small) changes but apparently git missed it. It doesn't attach the event listener immediately anymore and instead checks for the existence of the anchor elements before running the code